client/src/components/User.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import UserExtension from "./UserExtension.jsx";
+
+export default class User extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
+          <img
+            className="userPic"
+            src={this.props.data.userPicture}
+            alt="Logo"
+          />
+        </div>
+        <div className="userPro">
+          <div className="userName">
+            <span>{this.props.data.name}</span>
```

Comment:
Should use object destructuring for props.
```
const { name } = this.props.data;
```
to easily access the name value in-line.